### DISCUSSION TITLE
ayuda pokedex

### DISCUSSION THREAD

**Discussion by AreliBernal** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#512] [Q&A]
Hola chic@s
Me podrían ayudar y decirme como jalo otra info de la api, por ejemplo la altura, la verdad no entiendo nada y cada vez me hago mas bolas, muchas gracias.

**Comment by DanyVeneno** [comment]
Hola Areli mira aqui un ejemplo https://codepen.io/oryamne/pen/vYKXbgZ ahi te puedes apoyar espero te sirva

**Comment by OmarVenturaP** [comment]
![image](https://user-images.githubusercontent.com/99059639/160034719-0e30c386-bcb9-4d1c-a5bb-e1c361c6d3f0.png)


En el <a href="https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/blob/main/04%20-%20JS/programas/4.-pokedex.js">repositorio de la Practica</a> de JS, allí puedes ver el código y analizarlo. 

Si le das un console.log a los datos te muestra todos los que existen en la api, ya nada mas replicas el dato que está en el ejemplo y lo modificas con el que necesitas. Te puedes apoyar del live tambien.

**Comment by AreliBernal** [comment]
Ya vi varias veces el live y no le entiendo, es la primera vez que veo js así que me esta costando mucho, por eso quiero ver si alguien me ayuda en explicarme con bolitas y palitos como jalar otra info desde la api, pero muchas gracias por tu ayuda 😁

**Reply by LuisSantiago10** [reply]
https://www.youtube.com/watch?v=nFh7-HfODYY&t=784s&ab_channel=MakeAppswithDanny 

Es muy buen curso rápido , solo esta en ingles
